### PR TITLE
refactor: move port config inside a service dict

### PR DESCRIPTION
**Pull request by yprk**
- quick fix to ensure each service can run on a unique port until we have a new feature

### CODE REVIEW

File: collectington/config.py [hunk 1]
```diff
@@ -137,7 +133,7 @@ def validate_service(service_name, service):
 
 def validate_metrics_mapping(service_name, metrics_mapping):
     """Test that the metrics mapping of a service is valid."""
-    valid_metric_types = ["counter", "summary"]
+    valid_metric_types = ["counter", "summary", "histogram", "gauge"]
```
Comment (nathan-homex) [inline collectington/config.py:136]: **question**: are these other metric types that weren't mentioned earlier?

File: collectington/config.py [hunk 2]
```diff
@@ -85,9 +84,6 @@ def validate(config):
             "Invalid config: config should contain port, api_call_intervals, log_level, services"
         )
 
-    if not isinstance(config["port"], int):
-        raise ValueError("Invalid config: port should be an integer")
-
```
Comment (nathan-homex) [inline collectington/config.py:90]: **Change**: can you add this test to the `validate_service` section so we're still validating the port has the correct data type?

**Review by nathan-homex — CHANGES_REQUESTED** [review]
Looks good, just a small change needed

File: collectington/config.py (hunk 2 above)
Comment (yprk) [reply collectington/config.py:90]: done!

File: collectington/config.py (hunk 1 above)
Comment (yprk) [reply collectington/config.py:138]: these are the set of metric types that's been there and are currently being used.

**Review by nathan-homex — APPROVED** [review]
LGTM

**Pull request merged**